tests/framework/cli/test_cli.py

```diff
@@ -643,13 +629,13 @@ def test_run_with_params_in_config(
                 {"foo": "bar", "baz": "https://example.com"},
             ),
             ("foo:bar,baz:fizz buzz", {"foo": "bar", "baz": "fizz buzz"}),
-            ("foo:bar, foo : fizz buzz  ", {"foo": "fizz buzz"}),
```

Comment:
OmegaConf handles this case a bit differently. Instead of overwriting the value for the key, it keeps both. 
`{foo:"bar", foo:"fizz buzz"}`

Reply:
It would be breaking functionality if that behaviour suddenly changes. Can we make it such that the second provided argument overrides the first?

Reply:
Done in bdf35cf746e74741bf1706db3136f261181549a6